src/portal/lib/src/log/recent-log.component.ts

```diff
@@ -67,9 +67,9 @@ export class RecentLogComponent implements OnInit {
     }
 
     public doFilter(terms: string): void {
-        if (!terms) {
-            return;
-        }
+        // if (!terms) {
```

Comment:
it has conflict because of I has modify it  in branch master ,please update code and commit .

Reply:
![image](https://user-images.githubusercontent.com/29826789/63497517-7c6c9300-c4f6-11e9-815f-0cca00fae236.png)

Reply:
Bona fide reminder: Your code should use the same commit (just like the pull request you are submitting now)